app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+LOCATION = os.getenv("city")
```

Comment:
The 'LOCATION' variable is a constant and should be in uppercase. Also, it would be better to name it 'CITY' to reflect its purpose more clearly.